packages/fadbadml/fadbadml.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "FADBAD++ for OCaml"
+maintainer: [
+  "francois-bidet <fbidet@lix.polytechnique.fr>"
+  "ismailbennani <ismail.lahkim.bennani@ens.fr>"
+]
+authors: [
+  "francois-bidet <fbidet@lix.polytechnique.fr>"
+  "ismailbennani <ismail.lahkim.bennani@ens.fr>"
+]
+license: "CeCILL-C"
+homepage: "https://github.com/ismailbennani/FADBADml"
+bug-reports: "https://github.com/ismailbennani/FADBADml/issues"
+depends: [ "ocaml" { >= "4.08" } "ocamlfind" ]
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make "byte"] { !ocaml-native }
+  [make "opt"] { ocaml-native }
```

Comment:
The `ocaml-native` variable does not exist. I think what you meant is `ocaml:native`. However according the the `install` rule in your Makefile, these won't have any effect anyway as it relies on the `lib` rule. Those two lines should be replaced by a simple `[make "lib"]`.

Reply:
Done, thanks